### PR TITLE
Align Credentials part 4 (JavaScript)

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [Align Credentials for CiscoSpark, RSANetWitnessPacketsAndLogs, Giphy, PacketMail, CiscoCloudLock](https://jira-hq.paloaltonetworks.local/browse/CIAC-6946)

## Description
Aligns creds for:
- CiscoSpark
- RSANetWitnessPacketsAndLogs
- Giphy
- PacketMail
- CiscoCloudLock 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CiscoSpark/Integrations/CiscoSpark/CiscoSpark.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var apiKey = params.apiKey;
+var apiKey = params.apiKey || params.apikey_creds.password;
```
Comment (JudahSchwartz) [inline Packs/CiscoSpark/Integrations/CiscoSpark/CiscoSpark.js:1]: params.apiKey shouldnt take preference

File: Packs/CloudLock/Integrations/CiscoCloudLock/CiscoCloudLock.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ function sendRequest(commandData) {
         {
             Method: commandData.method,
             Headers: {
-                'Authorization': ['Bearer ' + params.token],
+                'Authorization': ['Bearer ' + (params.token || params.token_creds.password)],
```
Comment (JudahSchwartz) [inline Packs/CloudLock/Integrations/CiscoCloudLock/CiscoCloudLock.js:139]: same comment about priority

**Review by JudahSchwartz — APPROVED** [review]
nice

**Comment by dantavori** [comment]
force merging due to missing test validations failing for JS integrations without instance.

**Pull request merged**